GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -122,54 +132,58 @@ private void OnGuiCreateBundleSelect()
                 descriptionTextStyle);
+
             EditorGUILayout.Space();
+
+            EditorGUILayout.EndVertical();
+
             EditorGUILayout.Space();
-            EditorGUILayout.LabelField(
-                string.Format("Asset Bundle Browser version: {0}",
-                    AssetBundleBrowserClient.GetAssetBundleBrowserVersion()),
-                EditorStyles.wordWrappedLabel);
-            EditorGUILayout.Space();
+
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+
+            _playInstantSceneTreeTreeView.OnGUI(GUILayoutUtility.GetRect(position.width - 10, position.height - 200));
             EditorGUILayout.Space();
-            // Allow the developer to open the AssetBundles Browser if it is present, otherwise ask them to download it
-            if (AssetBundleBrowserClient.AssetBundleBrowserIsPresent())
+
+            EditorGUILayout.BeginHorizontal();
+
+            if (GUILayout.Button("Build AssetBundle"))
             {
-                if (GUILayout.Button("Open Asset Bundle Browser"))
+                try
                 {
-                    try
-                    {
-                        AssetBundleBrowserClient.DisplayAssetBundleBrowser();
-                    }
-                    catch (Exception ex)
-                    {
-                        DialogHelper.DisplayMessage(AssetBundleBrowserErrorTitle,
-                            ex.Message);
-
-                        throw;
-                    }
+                    QuickDeployConfig.AssetBundleFileName = EditorUtility.SaveFilePanel("Save AssetBundle", "", "", "");
+                    // TODO(audace): build the assetbundles
                 }
-            }
-            else
-            {
-                if (GUILayout.Button("Download Asset Bundle Browser from GitHub"))
+                catch (Exception ex)
                 {
-                    Application.OpenURL("https://github.com/Unity-Technologies/AssetBundles-Browser/releases");
+                    DialogHelper.DisplayMessage(AssetBundleBuildErrorTitle,
+                        ex.Message);
+                    throw;
                 }
+            }
+
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+
+            EditorGUILayout.EndVertical();
+        }
 
-                EditorGUILayout.Space();
-                EditorGUILayout.Space();
+        // TODO(audace): use this for building the scenes
+        private string[] GetEnabledSceneItemPaths()
+        {
+            var scenes = _playInstantSceneTreeTreeView.GetRows();
+            var scenePaths = new List<string>();
 
-                EditorGUILayout.Space();
-                if (GUILayout.Button("Open Asset Bundle Browser Documentation"))
+            foreach (var scene in scenes)
+            {
+                if ((scene as PlayInstantSceneTreeView.SceneItem).Enabled)
```

Comment:
I like "as", but if you're not going to check for null, just go ahead and do a regular cast. That way you get a casting exception rather than a null pointer exception.